WebHostLib/static/assets/tracker.js

```diff
@@ -70,10 +71,35 @@ window.addEventListener('load', () => {
         // the tbody and render two separate tables.
     });
 
-    document.getElementById('search').addEventListener('keyup', (event) => {
-        tables.search(event.target.value);
-        console.info(tables.search());
+    const searchBox = document.getElementById("search");
+    searchBox.value = tables.search();
+    const doSearch = () => {
+        tables.search(searchBox.value);
         tables.draw();
+    };
+    searchBox.addEventListener("keyup", doSearch);
+    window.addEventListener("keydown", (event) => {
+        if (event.ctrlKey && !event.altKey && !event.shiftKey && event.key === "f") {
```

Comment:
Overriding browser hotkeys is inadvisable. Also, since the search box is automatically focused when a user starts typing, this change seems less important. Especially if the search box is focused on page load.